uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -67,6 +67,116 @@ Feature: GGMQ-1
+                "target":"Pubsub"
+            },
+            "mapping3:":{
+                "topic":"topic/device2/#",
+                "source":"LocalMqtt",
+                "target":"Pubsub"
+            },
+            "mapping4:":{
+                "topic":"topic/with/prefix",
+                "source":"LocalMqtt",
+                "target":"Pubsub",
+                "targetTopicPrefix":"prefix/"
+            }
+        }
+    }
+}
+"""
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.Auth configuration to:
+    """
+{
+   "MERGE":{
+      "deviceGroups":{
+         "formatVersion":"2021-03-05",
+         "definitions":{
+            "MyPermissiveDeviceGroup":{
+               "selectionRule": "thingName: ${publisher}",
+               "policyName":"MyPermissivePolicy"
+            }
+         },
+         "policies":{
+            "MyPermissivePolicy":{
+               "AllowAll":{
+                  "statementDescription":"Allow client devices to perform all actions.",
+                  "operations":[
+                     "*"
+                  ],
+                  "resources":[
+                     "*"
+                  ]
+               }
+            }
+         }
+      }
+   }
+}
+    """
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.client.Mqtt5JavaSdkClient configuration to:
+    """
+{
+   "MERGE":{
+      "agentId":"aws.greengrass.client.Mqtt5JavaSdkClient",
+      "controlAddresses":"${mqttControlAddresses}",
+      "controlPort":"${mqttControlPort}"
+   }
+}
+    """
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.client.IpcClient configuration to:
+    """
+{
+   "MERGE": {
+       "accessControl": {
+           "aws.greengrass.ipc.pubsub": {
+               "aws.greengrass.client.IpcClient:pubsub:1": {
+                   "policyDescription":"access to pubsub topics",
+                   "operations": [ "aws.greengrass#SubscribeToTopic" ],
+                   "resources": [ "topic/to/pubsub", "topic/device1/humidity", "topic/device2/temperature", "prefix/topic/with/prefix" ]
+               }
+           }
+       },
+       "topicsToSubscribe": "topic/to/pubsub,topic/device1/humidity,topic/device2/temperature,prefix/topic/with/prefix"
+   }
+}
+    """
+    And I deploy the Greengrass deployment configuration
+    Then the Greengrass deployment is COMPLETED on the device after 300 seconds
+    Then I discover core device broker as "localMqttBroker" from "publisher"
+    And I connect device "publisher" on aws.greengrass.client.Mqtt5JavaSdkClient to "localMqttBroker"
+    And I wait 10 seconds
+    When I publish from "publisher" to "topic/to/pubsub" with qos 1 and message "Hello world"
+    Then the aws.greengrass.client.IpcClient log on the device contains the line "RECEIVED TOPIC=topic/to/pubsub, MESSAGE: Hello world" within 30 seconds
```

Comment:
this is fine for now, but ideally we shouldn't be verifying on logs since it's flaky (I realize this is the current pattern used in testing repo).  The way we did this before is setting up an HTTP server in the test steps and components would send their assertions to the server, I imagine we could add something similar in OTF